### ISSUE TITLE
Will mouse support be add ?

### ISSUE THREAD

**Issue by z826540272** [Maassoft/ColorControl#10]
I know it's technically possible, and mobile apps can do it

**Comment by Maassoft** [comment]
Yes, I will investigate this and probably add it in a future release.

**Comment by Maassoft** [comment]
I've added preliminary mouse support to the LG Remote Controller in the new release. It's far from perfect, but feel free to test it.
